README.md

```diff
@@ -64,20 +64,20 @@ siblings are immediately halted.
 
 ## Execution
 
-The process primitive is the `Execution`. To create (and start) an
+The process primitive is the `Execution`. To create (and enter) an
 `Execution`, use the `fork` function and pass it a generator. This
 simplest example waits for 1 second, then prints out "hello world" to
 the console.
 
 ``` javascript
-import { fork, timeout } from 'effection';
+import { enter, timeout } from 'effection';
```

Comment:
How about `run` instead of `enter`? Sounds a bit better to me.

Reply:
Yeah, `run` feels good, I had also considered `main()` since it's basically a single entry point into the top of your execution tree. The only reason I'd want to stick with `enter` or `main` is that really, this function is ideally meant to be called only once, where as `run` feels like maybe it's something you should do multiple times. Thoughts?

Reply:
Another idea that just occurred to me is that we could have a top-level `call` method, that way, you would just enter a tree with a top-level `call` on an operation.

Reply:
`main` sounds more clear if the intention is to create a single entry point. I agree that `run` and `call` both feel like something that happens a lot.

Reply:
`main` would feel more natural if we'd only ever call this once, like a `main` function in a C-program. But this is not necessarily the case.

Reply:
Agree, `main` works if it's expected to be unique in the process.

Reply:
I think that this is fundamentally the same operation as what you will do to create a sub child, so `spawn`, `call` or what have you. Let's resolve that discussion, and then that will inform what to do here.

Reply:
renamed to `spawn` for now.